personalize.module

```diff
@@ -1129,30 +1129,33 @@ function personalize_verify_agent($agent_name, $show_messages = TRUE) {
  * @param stdClass $agent_data
  *   (Optional) When provided then agent-specific transition steps can be
  *   included.  This is used for the "Scheduled" status.
+ * @param bool $include_all
+ *   (Optional) Include all transitions that are available.  If FALSE, only
+ *   those transitions that should be shown outside the campaign details
+ *   are returned.
```

Comment:
That's what I thought at first but it's not the case.  The campaign listing page is where we want some hidden and the campaign details page (for non-acquia lift agents) is where we need to show all options.  In the case of acquia lift agents we're actually fine leaving it the way it is due to the special snowflake handling of when a campaign is editable.  So it is called with a value of FALSE for the campaign list and with a value of TRUE in personalize.admin.campaign.inc.